### PR TITLE
Issue4492/migrate web

### PR DESCRIPTION
**Pull request by livarcocc**
Fixes https://github.com/dotnet/cli/issues/4492

cc @eerhardt @piotrpMSFT @mlorbetske @jgoshi @krwq

### CODE REVIEW

File: src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs [hunk 1]
```diff
@@ -6,5 +6,6 @@ namespace Microsoft.DotNet.ProjectJsonMigration
     internal class ConstantPackageNames
     {
         public const string CSdkPackageName = "Microsoft.NET.Sdk";
+        public const string CWebSdkPackageName = "Microsoft.NET.Sdk.Web";
```
Comment (piotrpMSFT) [inline src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs]: These names don't feel right. I know this PR isn't introducing the pattern, but we'll want to take another look.

File: src/Microsoft.DotNet.ProjectJsonMigration/ProjectTypeDetector.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    internal static class ProjectTypeDetector
+    {
+        public static string TryDetectProjectType(Project project)
+        {
+            string projectType = null;
+            if (IsWebProject(project))
+            {
+                projectType = "web";
```
Comment (piotrpMSFT) [inline src/Microsoft.DotNet.ProjectJsonMigration/ProjectTypeDetector.cs]: should this be an enum?

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build

File: src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs (hunk 1 above)
Comment (livarcocc) [inline src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs]: Fixed it.

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora23 x64 Debug Build

**Pull request merged**